app/index.js

```diff
@@ -21,6 +24,44 @@ const SWARM_HTTP_URL =
   (config && config.swarmHttpUrl) ||
   'https://onyx-storage.mainframe.com'
 
+const VersionCheckException = message => {
+  this.message = message
+  this.name = 'VersionCheckException'
+}
+
+const checkUpdates = () => {
+  fetch(VERSION_CHECK_URL)
+    .then(res => {
+      if (res.ok) {
+        return res.json()
+      } else {
+        throw new VersionCheckException('Error loading version')
+      }
+    })
+    .then(data => {
+      if (version !== data.version) {
+        dialog.showMessageBox(
+          mainWindow,
+          {
+            type: 'warning',
+            title: data.title,
```

Comment:
Might be good to set default values here for the keys other than `version`, ex here: `title: data.title || 'Update available',` in case it's not provided in the JSON.